oddbird/oddleventy
# Winging It 6

![](https://source.unsplash.com/featured/?cute,animal&sdfghjkl)

## Description
Winging it 6!

## Related Issue(s)
#507 

## Steps to test/reproduce
Check out [homepage](https://deploy-preview-565--oddleventy.netlify.app/#featured-posts) posts, then [posts page](https://deploy-preview-565--oddleventy.netlify.app/blog/), then [post single for winging it 6](https://deploy-preview-565--oddleventy.netlify.app/2024/04/18/winging-it-06/)

## Show me
![posts-page](https://github.com/oddbird/oddleventy/assets/1581694/80036307-9fab-43a5-ae51-94dba9346f4e)
![post-single](https://github.com/oddbird/oddleventy/assets/1581694/054e7b64-736a-489d-be87-59f449f9ffaa)
